mdm/common/src/main/java/com/gentics/mesh/core/db/CommonTx.java

```diff
@@ -0,0 +1,73 @@
+package com.gentics.mesh.core.db;
+
+import com.gentics.mesh.core.data.dao.BinaryDao;
+import com.gentics.mesh.core.data.dao.ContentDao;
+import com.gentics.mesh.core.data.dao.NodeDao;
+import com.gentics.mesh.core.data.dao.PersistingBranchDao;
+import com.gentics.mesh.core.data.dao.PersistingGroupDao;
+import com.gentics.mesh.core.data.dao.PersistingJobDao;
+import com.gentics.mesh.core.data.dao.PersistingLanguageDao;
+import com.gentics.mesh.core.data.dao.PersistingMicroschemaDao;
+import com.gentics.mesh.core.data.dao.PersistingProjectDao;
+import com.gentics.mesh.core.data.dao.PersistingRoleDao;
+import com.gentics.mesh.core.data.dao.PersistingSchemaDao;
+import com.gentics.mesh.core.data.dao.PersistingTagDao;
+import com.gentics.mesh.core.data.dao.PersistingTagFamilyDao;
+import com.gentics.mesh.core.data.dao.PersistingUserDao;
+
+/**
+ * A developer API of {@link Tx}.
+ * 
+ * @author plyhun
+ *
+ */
+public interface CommonTx extends Tx, TxEntityPersistenceManager {
+
+	/**
+	 * Return the current active common transaction.
+	 * 
+	 * @return Currently active transaction
+	 */
+	static CommonTx get() {
+		return (CommonTx) Tx.get();
+	}
+
+	@Override
+	PersistingUserDao userDao();
+
+	@Override
+	PersistingGroupDao groupDao();
+
+	@Override
+	PersistingRoleDao roleDao();
+
+	@Override
+	PersistingProjectDao projectDao();
+
+	@Override
+	PersistingLanguageDao languageDao();
+
+	@Override
+	PersistingJobDao jobDao();
+
+	@Override
+	PersistingTagFamilyDao tagFamilyDao();
+
+	@Override
+	PersistingTagDao tagDao();
+
+	@Override
+	PersistingBranchDao branchDao();
+
+	@Override
+	PersistingMicroschemaDao microschemaDao();
+
+	@Override
+	PersistingSchemaDao schemaDao();
+
+	BinaryDao binaryDao();
+
+	NodeDao nodeDao();
```

Comment:
Add @Override

Reply:
This and the unchanged DAO accessors below have been removed because of being unneeded, as of now.